migrations/20170913153311-remove-unique-contraint-for-botaction.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+'use strict'; // eslint-disable-line strict
+
+module.exports = {
+  up(qi) {
```

Comment:
have you run this multiple times locally? when the constraint exists then again when it doesn't?

Reply:
I ran this test:
branch master: npm start - pass
branch duplicateBotActions: npm start - add migration, pass
branch duplicateBotActions - delete migration from db: npm start - run migration again, pass

Are there anyone tests you think I should run to make sure it works?

Reply:
ok